### PR TITLE
chore(highlights): Fix e2e tests

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/app/scripts/superdesk-archive/views/list.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	<div class="button-stack right-stack">   
 	    <div class="navbtn dropdown strict highlights-dropdown" 
             dropdown sd-package-highlights-dropdown 
-            ng-if="type === 'archive' && selected.desk">
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-archive/views/list.html]: @petrjasek  At line 21 is still used selected.desk and it looks like is not set correctly when current desk selection is changed. I was not able to fix it.

**Comment by petrjasek** [comment]
seems like there are some js formatting issues

File: client/app/scripts/superdesk-archive/views/list.html (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk-archive/views/list.html]: I think you can fix it if you add `$scope.selected = active;` to [controller](https://github.com/superdesk/superdesk/blob/master/client/app/scripts/superdesk-archive/controllers/list.js#L107-L114)

File: client/spec/helpers/highlights.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+
+'use strict';
+
+var openUrl = require('./helpers/utils').open;
```
Comment (petrjasek) [inline client/spec/helpers/highlights.js]: here it should be `require('./utils').open` it's in helpers folder

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**